networking/hardware_networks/using-pod-level-bonding.adoc

```diff
@@ -8,11 +8,17 @@ toc::[]
 
 
 :FeatureName: Bond Container Network Interface (CNI)
+include::snippets/technology-preview.adoc[leveloffset=+1]
 
 Bonding at the pod level is vital to enable workloads inside pods that require high availability and more throughput. With pod-level bonding, you can create a bond interface from multiple single root I/O virtualization (SR-IOV) virtual function interfaces in a kernel mode interface. The SR-IOV virtual functions are passed into the pod and attached to a kernel driver.
 
 One scenario where pod level bonding is required is creating a bond interface from multiple SR-IOV virtual functions on different physical functions. Creating a bond interface from two different physical functions on the host can be used to achieve high availability and throughput at pod level.
 
+[NOTE]
+====
+The current functionality of Bond CNI is available only in active-backup mode - for further details, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2037214[*BZ#2037214*].
```

Comment:
```suggestion
The current functionality of Bond CNI is available only in active-backup mode. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2037214[*BZ#2037214*].
```
Inline links should be introduced with "For more information" [1]

[1] https://docs.engineering.redhat.com/display/CCSENABLE/Inline+links